action/protocol/multichain/subchain/protocol.go

```diff
@@ -29,17 +27,16 @@ const ProtocolID = "multi-chain_sub-chain"
 
 // Protocol defines the protocol to handle multi-chain actions on sub-chain
 type Protocol struct {
-	chainID      uint32
-	mainChainAPI explorer.Explorer
-	sf           factory.Factory
+	chainID uint32
+	// TODO: explorer dependency deleted, need to add api related params
```

Comment:
Also, can we add "https://github.com/iotexproject/iotex-core/pull/1085" so that we could refer to the deleted code, when we need to revive this functionality.

Same for the bellow

Reply:
sure